### PR TITLE
Submission week02

### PR DESCRIPTION
**Pull request by fifo36**

### CODE REVIEW

File: week02/833_Find_And_Replace_in_String.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
+//833. Find And Replace in String
+//TC: O(N * |S| + |S|)   -> N := indices.size()
+//MC: O(N + |S|)
+
+class Solution {
+public:
+    string findReplaceString(string s, vector<int>& indices, vector<string>& sources, vector<string>& targets) {
+        int sz = s.size(), isValid[sz];
```
Comment (Sadman007) [inline week02/833_Find_And_Replace_in_String.cpp:8]: DO NOT use variable-sized array. It's a red flag. Even some compiler does not even let you compile this code.

Always use `vector` in these cases.

**Review by Sadman007 — APPROVED** [review]
Hi @fifo36 

Nice work. You can merge this branch with `master/main` now.

**Comment by Sadman007** [comment]
Hi @fifo36 

I have completed my review. You can find (if any) suggestions in this thread. Feel free to merge this branch with `master/main`. You should keep this branch too for future reference.

File: week02/833_Find_And_Replace_in_String.cpp (hunk 1 above)
Comment (fifo36) [reply week02/833_Find_And_Replace_in_String.cpp:8]: Wow! Thanks, vai. I will practice it later.

**Pull request closed without merging**